### PR TITLE
November notes

### PR DESCRIPTION
**Pull request by thomasp-ms**
## Purpose

* Release notes for current iteration

## What to Check
* Please check I didn't mis-represent anything
* Please check I didn't forget anything important

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,33 @@
...
+
+Here are the new features.
+
+<!-- ### Provisioning
+-  -->
+
+### Documentation
+- We have introduced a new [Troubleshooting Guide](./docs/tsg.md) to help you troubleshoot common issues. We will keep adding to it as we become aware of more common issues.
+
+### FL Experience
```
Comment (jfomhover) [inline CHANGELOG.md:15]: could we move this up in the release notes? or maybe mention on the top that the main value add of this release are the 3 examples?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,33 @@
...
+- We fixed a bug causing the contents of the `config.json` file to take precedence over the CLI arguments (subscription id, resource group, workspace name) when submitting a job. Now, the CLI arguments takes precedence over the `config.yaml` file, which itself takes precedence over the `config.json` file.
+- We have added 3 industry-relevant examples. The examples include 2 jobs. The first one (generic) downloads some public data, partitions them, and uploads them to the silos. The second job (example-specific) trains a model using FL. The 3 examples address the following topics:
```
Comment (jfomhover) [inline CHANGELOG.md:17]: let's put this on top :)

File: CHANGELOG.md (hunk 1 above)
Comment (thomasp-ms) [reply CHANGELOG.md:12]: done

File: CHANGELOG.md (hunk 2 above)
Comment (thomasp-ms) [reply CHANGELOG.md:13]: done

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,33 @@
...
+  - [named entity recognition](./docs/real-world-examples/ner.md) (NLP example);
+  - [pneumonia detection](./docs/real-world-examples/pneumonia.md) (medical imaging example). 
+- We fixed a bug causing the contents of the `config.json` file to take precedence over the CLI arguments (subscription id, resource group, workspace name) when submitting a job. Now, the CLI arguments takes precedence over the `config.yaml` file, which itself takes precedence over the `config.json` file.
+
+### Documentation
+- We have introduced a new [Troubleshooting Guide](./docs/tsg.md) to help you troubleshoot common issues. We will keep adding to it as we become aware of more common issues.
+
+### Repository structure
+- We improved the [documentation home page](./docs/README.md) by adding some pictures and introducing the industry-relevant examples.
+- We revisited our CI/CD processes for better agility (enable concurrent jobs, accommodate token expiration issue).
```
Comment (garg-amit) [inline CHANGELOG.md:24]: how about rephrasing this to 
`Introduced a new ...`
`Improved the...`
n everywhere?
Just to maintain consistency with the prev. release notes

**Review by garg-amit — APPROVED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (thomasp-ms) [reply CHANGELOG.md]: good call

**Pull request merged**